apps/opik-backend/src/main/java/com/comet/opik/api/resources/v1/priv/AutomationRuleEvaluatorsResource.java

```diff
@@ -0,0 +1,168 @@
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.headers.Header;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.parameters.RequestBody;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import jakarta.inject.Inject;
+import jakarta.inject.Provider;
+import jakarta.validation.Valid;
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
+import jakarta.ws.rs.Consumes;
+import jakarta.ws.rs.DELETE;
+import jakarta.ws.rs.DefaultValue;
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.POST;
+import jakarta.ws.rs.PUT;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.PathParam;
+import jakarta.ws.rs.Produces;
+import jakarta.ws.rs.QueryParam;
+import jakarta.ws.rs.core.Context;
+import jakarta.ws.rs.core.MediaType;
+import jakarta.ws.rs.core.Response;
+import jakarta.ws.rs.core.UriInfo;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+
+import java.net.URI;
+import java.util.UUID;
+
+@Path("/v1/private/automation/evaluators/project/{projectId}")
+@Produces(MediaType.APPLICATION_JSON)
+@Consumes(MediaType.APPLICATION_JSON)
+@Timed
+@Slf4j
+@RequiredArgsConstructor(onConstructor_ = @Inject)
+@Tag(name = "Automation rule evaluators", description = "Automation rule evaluators resource")
+public class AutomationRuleEvaluatorsResource {
+
+    private final @NonNull AutomationRuleEvaluatorService service;
+    private final @NonNull Provider<RequestContext> requestContext;
+
+    @GET
+    @Operation(operationId = "findEvaluators", summary = "Find Evaluators", description = "Find Evaluators", responses = {
+            @ApiResponse(responseCode = "200", description = "Evaluators resource", content = @Content(schema = @Schema(implementation = AutomationRuleEvaluator.AutomationRuleEvaluatorPage.class)))
+    })
+    @JsonView(AutomationRuleEvaluator.View.Public.class)
+    public Response find(@PathParam("projectId") UUID projectId,
+            @QueryParam("page") @Min(1) @DefaultValue("1") int page,
+            @QueryParam("size") @Min(1) @DefaultValue("10") int size) {
+
+        String workspaceId = requestContext.get().getWorkspaceId();
+        log.info("Looking for automated evaluators for project id '{}' on workspaceId '{}' (page {})", projectId,
+                workspaceId, page);
+        Page<AutomationRuleEvaluator> definitionPage = service.find(page, size, projectId, workspaceId);
+        log.info("Found {} automated evaluators for project id '{}' on workspaceId '{}' (page {}, total {})",
+                definitionPage.size(), projectId, workspaceId, page, definitionPage.total());
+
+        return Response.ok()
+                .entity(definitionPage)
+                .build();
+    }
+
+    @GET
+    @Path("/evaluator/{evaluatorId}")
```

Comment:
The top level path above is: `@Path("/v1/private/automation/evaluators/project/{projectId}")`

This will be combined and give you: `/v1/private/automation/evaluators/project/{projectId}/evaluator/{evaluatorId}`. 

This is not right, you shouldn't duplicate `evaluators` and `evaluator`.